db/migrations/20240529151056_Page.sql

```diff
@@ -0,0 +1,9 @@
+-- Ensure the UUID extension is enabled
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE Page (
+    page_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    user_id Uuid UNIQUE,
+    url_id Uuid UNIQUE,
+    url VARCHAR(255) NOT NULL
```

Comment:
Another thing I think we want to add to this table/migration is a UNIQUE index over the `user_id` and `url` columns

That way Postgres can help make sure that for any user they don't have multiple `Pages` that point to the same URL.
But it will allow different users to point save the same URL without issues!

Happy to help with the syntax for this if you run into issues!